### ISSUE TITLE
Non-Annotated MRI Frame Warning

### ISSUE THREAD

**Issue by FMCalisto** [MIMBCD-UI/prototype-breast-screening#99]
The [Magnetic Resonance Imaging (MRI)](https://en.wikipedia.org/wiki/Magnetic_resonance_imaging) technology is a type of scan that allows the visualization of several slices, framing layer-by-layer and a picture taken of each slice. This technology can help diagnose the lesions' progress across the set of slices. In our [annotations dataset](https://github.com/MIMBCD-UI/dataset-annotations), we need to provide complete information regarding the existence of injuries in all frames that a lesion is presented. Therefore, we can not accept frames where there is evidence of lesion, but the clinician passed through it missing the lesion annotation.

With this issue, our goal is to surpass the problem of misannotation a frame on a set of slices. The idea is, whenever the system starts to receive at, for instance, slice `s = 121` and finish at the slice `s = 134` it needs to guarantee that **ALL** slices between those are annotated. In the following image ([lucidchart.com](https://www.lucidchart.com/invitations/accept/ed31256e-dfae-453c-9bc7-5825c82e8dc0)), we can see if the slice `s = 128` is not annotated, the system will trigger a warning when the clinician press the *saving* button.

![warning_mri_missing_frames](https://user-images.githubusercontent.com/1298239/52215668-72ea4900-288c-11e9-9dc3-d72a6f57a4af.png)

In short, there is the (high) probability of a clinician to jump between slices. For that reason, we need to guarantee that the system warns the clinician to fix that problem. The solution to it is, each time a clinician press the *saving* button or closes (the behavior is the same as the last) the respective patient, we will warn the clinician.

### Related Issues

- [Fix: MRI Jumping Slices](https://github.com/MIMBCD-UI/prototype-breast-screening/issues/100) #100
